demos/storybook/stories/drawer/_module.stories.ts

```diff
@@ -113,4 +113,5 @@ storiesOf(`${COMPONENT_SECTION_NAME}/Drawer`, module)
     .addDecorator(storyWrapper())
     .addParameters({ ...STORY_PARAMS, notes: { markdown: getReadMe('Drawer.md') } })
     .add(README_STORY_NAME, getReadMeStory)
-    .add('within a Drawer Layout', withinDrawerLayout);
+    .add('within a Drawer Layout', withinDrawerLayout)
```

Comment:
Why do we have two different storiesOf calls for the Drawer?

Reply:
All the Drawer stories have a special wrapper (`drawerWrapper`) around it that'll shift canvas content onto the left-side of the screen.   This wrapper doesn't apply to the DrawerLayout stories and will mess up its styles.